schema/user.js

```diff
@@ -29,7 +30,11 @@ const userResolvers = {
         v1AccessToken,
         v2AccessToken
       );
-      return { userSettings };
+      const currencySettings = await dataSources.walloraAPI.getCurrencySettings(
+        v1AccessToken,
```

Comment:
Can we find out a way to provide these 2 access tokens to the datasource methods through other means rather than passing explicitly in all methods? (to improve readability of code)